<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>595</width>
    <height>426</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="palette">
       <palette>
        <active>
         <colorrole role="Base">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>240</red>
            <green>240</green>
            <blue>240</blue>
           </color>
          </brush>
         </colorrole>
        </active>
        <inactive>
         <colorrole role="Base">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>240</red>
            <green>240</green>
            <blue>240</blue>
           </color>
          </brush>
         </colorrole>
        </inactive>
        <disabled>
         <colorrole role="Base">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>240</red>
            <green>240</green>
            <blue>240</blue>
           </color>
          </brush>
         </colorrole>
        </disabled>
       </palette>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="page">
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QTableView" name="tableView">
          <property name="alternatingRowColors">
           <bool>true</bool>
          </property>
          <property name="gridStyle">
           <enum>Qt::NoPen</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_2">
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QListWidget" name="create_set">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>60</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>125</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="baseSize">
           <size>
            <width>60</width>
            <height>0</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Calibri</family>
            <pointsize>10</pointsize>
            <italic>true</italic>
           </font>
          </property>
          <property name="autoFillBackground">
           <bool>true</bool>
          </property>
          <property name="frameShape">
           <enum>QFrame::HLine</enum>
          </property>
          <property name="editTriggers">
           <set>QAbstractItemView::DoubleClicked|QAbstractItemView::EditKeyPressed|QAbstractItemView::SelectedClicked</set>
          </property>
          <property name="iconSize">
           <size>
            <width>48</width>
            <height>48</height>
           </size>
          </property>
          <property name="viewMode">
           <enum>QListView::IconMode</enum>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QListWidget" name="all_sets">
          <property name="frameShape">
           <enum>QFrame::HLine</enum>
          </property>
          <property name="iconSize">
           <size>
            <width>48</width>
            <height>48</height>
           </size>
          </property>
          <property name="viewMode">
           <enum>QListView::IconMode</enum>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>595</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_file">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="actionSave"/>
    <addaction name="separator"/>
    <addaction name="action_4"/>
    <addaction name="action_5"/>
   </widget>
   <widget class="QMenu" name="menu_service">
    <property name="title">
     <string>Сервис</string>
    </property>
    <addaction name="action_importExcel"/>
    <addaction name="action_Excel_2"/>
    <addaction name="separator"/>
    <addaction name="action_3"/>
   </widget>
   <widget class="QMenu" name="menu_help">
    <property name="title">
     <string>Справка</string>
    </property>
    <addaction name="action_6"/>
    <addaction name="action_7"/>
   </widget>
   <addaction name="menu_file"/>
   <addaction name="menu_service"/>
   <addaction name="menu_help"/>
  </widget>
  <widget class="QToolBar" name="toolBar_editor">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_editorTool"/>
   <addaction name="separator"/>
   <addaction name="actionSave"/>
   <addaction name="separator"/>
   <addaction name="action_prepare"/>
   <addaction name="separator"/>
   <addaction name="action_calcilate"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_editorTool">
   <property name="text">
    <string>Редактор</string>
   </property>
   <property name="toolTip">
    <string>Открыть редактор</string>
   </property>
  </action>
  <action name="action_importExcel">
   <property name="text">
    <string>Импорт Excel</string>
   </property>
  </action>
  <action name="action_Excel_2">
   <property name="text">
    <string>Экспорт Excel</string>
   </property>
  </action>
  <action name="action_3">
   <property name="text">
    <string>Настройки подключения к БД</string>
   </property>
  </action>
  <action name="action_4">
   <property name="text">
    <string>Открыть набор</string>
   </property>
  </action>
  <action name="action_5">
   <property name="text">
    <string>Создать новый</string>
   </property>
  </action>
  <action name="action_6">
   <property name="text">
    <string>О программе</string>
   </property>
  </action>
  <action name="action_7">
   <property name="text">
    <string>Руководство пользователя</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Сохранить</string>
   </property>
   <property name="toolTip">
    <string>Сохранить изменения</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="action_prepare">
   <property name="text">
    <string>Подготовить</string>
   </property>
   <property name="toolTip">
    <string>Очистить данные от шума и стандартизировать их</string>
   </property>
  </action>
  <action name="action_calcilate">
   <property name="text">
    <string>Рассчитать</string>
   </property>
   <property name="toolTip">
    <string>Начать разделени гидрографа речного стока</string>
   </property>
  </action>
  <action name="action_plot">
   <property name="text">
    <string>Диаграммы</string>
   </property>
   <property name="toolTip">
    <string>Диаграммы смешения</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
